primitives/state-machine/src/backend.rs

```diff
@@ -26,6 +26,12 @@ use trie::{
 	trie_types::{TrieDBMut, Layout},
 };
 use codec::{Encode, Codec};
+use primitives::storage::{ChildInfo, OwnedChildInfo};
+
+pub(crate) type StorageTuple = (
```

Comment:
This whole type would look much better as a struct with another struct for the second `HashMap` value